perf/lua/box_select.lua

```diff
@@ -1,27 +1,34 @@
 --
 -- The test measures run time of various space select operations.
 --
--- Output format:
--- <test-case> <run-time-nanoseconds>
+-- Output format (console):
+-- <test-case> <iterations-per-second>
```

Comment:
I would ask @locker to glance, because he is the author of the benchmark.

Reply:
Volodya gave LGTM, so this can be closed.

https://github.com/tarantool/tarantool/pull/10061#pullrequestreview-2098975704